### PR TITLE
Added button

### PR DESCRIPTION
**Pull request by anshikaa12**
>added button to the documentation
buttondoc

### CODE REVIEW

File: components/buttons/button.css [hunk 1]
```diff
@@ -41,29 +41,35 @@ color:var(--complement-btn-text-color);
 .btn-primary-outline{
     background-color:transparent;
     outline:1px solid var(--primary-color);
-    color:var(--complement-btn-text-color);
+    color:var(--text-color);
```
Comment (aryan-13) [inline components/buttons/button.css:44]: you can select all the classes at once and then give them the desired color at once, Makes it easier to change the color of all the buttons in one go

Comment (anshikaa12) [reply components/buttons/button.css:44]: Actually, I have different colors for different classes. 
primary for primary-btn class, accent for accent-btn class that is why cannot combine them all together!

Comment (aryan-13) [reply components/buttons/button.css:44]: Okay

File: documentation/documentation.html [hunk 2]
```diff
@@ -291,9 +291,199 @@ <h4 class="card-title">Sample Card</h4>
...
+                    <br>
+                    You can add various colors according to your color scheme using classes like <span
+                        class=prime-color-text>btn-primary, btn-secondary, btn-accent.</span>
+                    <br>
+                    <br>
+                    <code class="main-code">
+                        btn:
+                    </code>
+                    To add padding and font size
+                    <br>
```
Comment (junaidshaikh-js) [inline documentation/documentation.html:318]: Don't use `br` element to add the line break. Instead, use flexbox to arrange the elements.

**Review by junaidshaikh-js — COMMENTED** [review]
Everything else looks good.

File: components/buttons/button.css [hunk 3]
```diff
@@ -113,5 +120,10 @@ color:var(--complement-btn-text-color);
     background-color:var(--primary-color);
     opacity:0.7;
    }
-  
+  /************************/
+.link-btn{
+    text-decoration:none;
+    border-radius: .5rem;
```
Comment (paarthbajaj) [inline components/buttons/button.css:126]: Please go constant about units. Use the same units if possible.
For example here: The unit used is rem and in line 105, it is in %. Please correct me if I said something wrong.

Comment (anshikaa12) [reply components/buttons/button.css:126]: will see to it !

Comment (paarthbajaj) [reply components/buttons/button.css:126]: Okay great. Everything else is good.

**Comment by ajit-sharmaGH** [comment]
Looks cool 😎

**Pull request merged**